### DISCUSSION TITLE
New Release?

### DISCUSSION THREAD

**Discussion by tamarinkeisari** [projectnessie/nessie#8862] [General]
When will the new release be published?

**Comment by dimas-b** [comment]
Hi @tamarinkeisari , 

At this stage there is no regular release cadence for Nessie.

Is there a particular feature of a bugfix that you would like to see released?

It might be easier (certainly more interactive :) ) to discuss releases in our Zulip chat. The join link is on projectnessie.org.

**Reply by tamarinkeisari** [reply]
Hi, if you could please update the docker image to include the changes of issue number #8849. Thank you

**Reply by dimas-b** [reply]
#8849 is part of Nessie v. [0.91.1](https://github.com/projectnessie/nessie/releases/tag/nessie-0.91.1)
